backtrader/brokers/ccxtbroker.py

```diff
@@ -96,7 +126,19 @@ def sell(self, owner, data, size, price=None, plimit=None,
         return self._submit(owner, data, exectype, 'sell', size, price, kwargs)
 
     def cancel(self, order):
-        return self.store.cancel_order(order['id'])
+        oID = order.ccxt_order['id']
+        # check first if the order has already been filled otherwise an error
+        # might be raised if we try to cancel an order that is not open.
+        ccxt_order = self.store.fetch_order(oID)
+        if ccxt_order['status'] == 'closed':
+            return order
+
+        ccxt_order = self.store.cancel_order(oID)
+        if ccxt_order['status'] == 'canceled':
```

Comment:
In case of binance of binance this gives an `key error` as the canceled order no longer contains a `status` field.